sql/core/src/test/scala/org/apache/spark/sql/execution/columnar/ColumnTypeSuite.scala

```diff
@@ -79,6 +80,8 @@ class ColumnTypeSuite extends SparkFunSuite with Logging {
     checkActualSize(MAP_TYPE, Map(1 -> "a"), 4 + (8 + 8 + 8 + 8) + (8 + 8 + 8 + 8))
     checkActualSize(STRUCT_TYPE, Row("hello"), 28)
     checkActualSize(CALENDAR_INTERVAL, new CalendarInterval(0, 0, 0), 4 + 4 + 8)
+    checkActualSize(INT, Period.ofMonths(Int.MaxValue).normalized(), 4)
+    checkActualSize(LONG, Duration.ofDays(106751991), 8)
```

Comment:
hmm, why `INT` and `LONG` column types? I wonder what do you construct here:
```scala
val proj = UnsafeProjection.create(Array[DataType](columnType.dataType))
```
Interval types or Int/Long?